tests/Microsoft.Identity.Test.LabInfrastructure/LabUser.cs

```diff
@@ -39,19 +39,37 @@ public class LabUser
         [JsonProperty("labname")]
         public string LabName { get; set; }
 
-        public FederationProvider FederationProvider { get; set; }
+        //TODO: request the lab to add the federation provider to the lab user response
+        public FederationProvider FederationProvider
```

Comment:
@bgavrilMS they seem to have removed the federation provider from the lab response.